### PR TITLE
fixed error when using get with empty namespace

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

## Description
k8s api rest methods not working properly with empty namespaces, so changed get to list

## Issue
part of https://github.com/networkservicemesh/sdk-k8s/issues/252

### CODE REVIEW

File: pkg/registry/etcd/ns_server.go [hunk 1]
```diff
@@ -58,11 +58,10 @@ func (n *etcdNSRegistryServer) Register(ctx context.Context, request *registry.N
 		metav1.CreateOptions{},
 	)
 	if apierrors.IsAlreadyExists(err) {
-		var exist *v1.NetworkService
-		exist, err = n.client.NetworkservicemeshV1().NetworkServices("").Get(ctx, request.Name, metav1.GetOptions{})
+		var ns *v1.NetworkService
+		ns, err = n.findNetworkService(ctx, request)
 		if err == nil {
-			exist.Spec = *(*v1.NetworkServiceSpec)(request)
```
Comment (denis-tingaikin) [inline pkg/registry/etcd/ns_server.go:64]: Why is this line removed?

File: pkg/registry/etcd/nse_server.go [hunk 2]
```diff
@@ -62,11 +62,10 @@ func (n *etcdNSERegistryServer) Register(ctx context.Context, request *registry.
 		metav1.CreateOptions{},
 	)
 	if apierrors.IsAlreadyExists(err) {
-		var exist *v1.NetworkServiceEndpoint
-		exist, err = n.client.NetworkservicemeshV1().NetworkServiceEndpoints("").Get(ctx, request.Name, metav1.GetOptions{})
+		var nse *v1.NetworkServiceEndpoint
+		nse, err = n.findNetworkServiceEndpoint(ctx, request)
 		if err == nil {
-			exist.Spec = *(*v1.NetworkServiceEndpointSpec)(request)
```
[the remark this answers is not in the document]
Comment (Mixaster995) [reply pkg/registry/etcd/nse_server.go:68]: fixed

File: pkg/registry/etcd/ns_server.go (hunk 1 above)
Comment (Mixaster995) [reply pkg/registry/etcd/ns_server.go:64]: fixed
